python/ray/tune/logger.py

```diff
@@ -219,17 +219,16 @@ def on_result(self, result):
 
         for attr, value in flat_result.items():
             full_attr = "/".join(path + [attr])
-            if type(value) in VALID_SUMMARY_TYPES and not np.isnan(value):
+            if isinstance(value, VALID_SUMMARY_TYPES) and not np.isnan(value):
```

Comment:
this needs to cast VALID_SUMMARY_TYPES to tuple (tests are failing)

Reply:
```suggestion
            if (isinstance(value, tuple(VALID_SUMMARY_TYPES))
                and not np.isnan(value)):
```